### ISSUE TITLE
Clicking on city links is broken in iOS Firefox

### ISSUE THREAD

**Issue by qualitymanifest** [qualitymanifest/foambook#18]
While attempting to navigate the query menu, clicking on a panel will expand the panel but clicking on an item in the panel does not navigate to that link. This only happens on iOS Firefox.

**Comment by qualitymanifest** [comment]
Bizarrely, it is also broken when using a mobile device simulator in chromium devtools. When it fails, if you inspect what's going on in [`parseQueryString`](https://github.com/qualitymanifest/foambook/blob/master/imports/utils/queryFunctions.js#L10), `window.location.search` is what it should be, but `Array.from(queryString)` is empty.

If you refresh the page while in full screen mode, then change to the mobile simulator, it works.

If you refresh the page while in the mobile simulator, and then switch to full screen mode, it fails in full screen mode. This is the only way you can get it to fail in full screen mode.

**Comment by qualitymanifest** [comment]
This issue can be duplicated in the console, only when the mobile simulator is open:
```js
let params = new URLSearchParams("?city=PITTSBURGH&state=PA")
console.log(params.get("city")) // this works regardless, so it's not an issue of URLSearchParams not picking up the params
console.log(Array.from(params)) // Works on desktop, empty on mobile simulator
console.log(...params) // Works on desktop and mobile simulator
```
So it looks to be an issue with `Array.from`, but what? I don't think it's an issue of `Array.from` being overwritten somewhere, because when you console.log it, it still takes you to the native code.

Repeating the above in the console with a different website open works without issue.

**Comment by qualitymanifest** [comment]
Closed in #476d838
